tests/acceptance/features/apiSpacesDavOperation/moveByFileId.feature

```diff
@@ -150,6 +150,31 @@ Feature: moving/renaming file using file id
       | /dav/spaces/<<FILEID>>            |
 
 
+  Scenario Outline: Move a file from sub-folder to root folder inside Shares space
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "/folder"
+    And user "Alice" has created folder "folder/sub-folder"
+    And user "Alice" has uploaded file with content "some data" to "/folder/sub-folder/test.txt"
+    And we save it into "FILEID"
+    And user "Alice" has shared folder "/folder" with user "Brian" with permissions "all"
+
+    When user "Brian" moves a file "Shares/folder/sub-folder/test.txt" into "Shares/folder" inside space "Shares" using file-id path "<dav-path>"
```

Comment:
```suggestion
    When user "Brian" moves a file "Shares/folder/sub-folder/test.txt" into "folder" inside space "Shares" using file-id path "<dav-path>"
```
do we specificallu need to define `Shares/folder` ? since we know its inside space `Shares`.

Reply:
Without `shares/` it Cannot find share mountpoint id